Makefile

```diff
@@ -569,6 +569,8 @@ test_web_symfony_34:
 	php tests/Frameworks/Symfony/Version_3_4/bin/console cache:clear --no-warmup --env=prod
 	$(call run_tests,tests/Integrations/Symfony/V3_4)
 test_web_symfony_40:
+	# Trick to have symfony 4.0 update process not to fail because of an error related to monolog dependencies.
+	$(COMPOSER) --working-dir=tests/Frameworks/Symfony/Version_4_0 update --no-scripts
 	$(COMPOSER) --working-dir=tests/Frameworks/Symfony/Version_4_0 update
```

Comment:
Locally this still fails for me. The `--no-scripts` line succeeds, then this one fails.

Reply:
[This comment](https://github.com/symfony/recipes/issues/639#issuecomment-647379828) makes it seem like maybe we are doing something wrong with recipes.